src/dev.sh

```diff
@@ -304,13 +304,21 @@ function package ()
         tar_name="${agent_pkg_name}.tar.gz"
         echo "Creating $tar_name in ${LAYOUT_DIR}"
         tar -czf "${tar_name}" -C ${LAYOUT_DIR} .
+        echo "Calculate SHA256 checksum for $tar_name"
+        if [[ ("$PLATFORM" == "linux") ]]; then
+            sha256sum "${tar_name}" | cut -b 1-64 > ./Checksum_SHA256.txt
```

Comment:
@ericsciple also, should we not have 

```
sha256sum "${tar_name}" | cut -b 1-64 > ./${tar_name}.sha256
```

rather than 

```
sha256sum "${tar_name}" | cut -b 1-64 > ./Checksum_SHA256.txt
```

Surely there is more than one binary file, so having one `Checksum_SHA256.txt` file will not work?